systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,359 @@
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+        resourceManager.createResource(extensionContext, adminClientJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(createAdminName, NAMESPACE, topicsCountBelowQuota);
+        String createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(createPodName, "All topics created");
+
+        // List 'topicsCountBelowQuota' topics
+        KafkaAdminClient adminClientListJob = new KafkaAdminClient.Builder()
+                .withAdminName(listAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.LIST_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+        resourceManager.createResource(extensionContext, adminClientListJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(listAdminName, NAMESPACE, 0);
+        String listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(listPodName, topicNamePrefix + "-" + (topicsCountBelowQuota - 1));
+        JobUtils.deleteJobWithWait(NAMESPACE, listAdminName);
+
+        // Delete 'topicsCountBelowQuota' topics
+        adminClientJob = adminClientJob.toBuilder()
+                .withAdminName(deleteAdminName)
+                .withTopicOperation(AdminClientOperations.DELETE_TOPICS.toString())
+                .build();
+        resourceManager.createResource(extensionContext, adminClientJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(deleteAdminName, NAMESPACE, 0);
+        String deletePodName = kubeClient().listPods("job-name", deleteAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(deletePodName, "Successfully removed all " + topicsCountBelowQuota);
+
+        // List topics after deletion
+        resourceManager.createResource(extensionContext, adminClientListJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(listAdminName, NAMESPACE, 0);
+        listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        String afterDeletePodLogs = kubeClient().logs(listPodName);
+        assertThat(afterDeletePodLogs.contains(topicNamePrefix), is(false));
+        assertThat(afterDeletePodLogs, not(containsString(topicNamePrefix)));
+
+        JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        JobUtils.deleteJobWithWait(NAMESPACE, listAdminName);
+        JobUtils.deleteJobWithWait(NAMESPACE, deleteAdminName);
+    }
+
+    String getAdminClientConfig(String kafkaUsername) {
```

Comment:
```suggestion
    private String getAdminClientConfig(String kafkaUsername) {
```
but this method should be in Utils classes in my opinion

Reply:
It's quite related to tests only, as it returns just needed sasl part. It's only a helper method.